fetch.bs

```diff
@@ -4141,16 +4131,12 @@ run these steps:
   </ol>
 
  <li>
-  <p>Otherwise, if <var>object</var> is an open-ended dictionary, then for each
-  <var>header</var> in <var>object</var>, run these substeps:
+  <p>Otherwise, if <var>object</var> is a record, then for each
+  mapping (<var>key</var>, <var>value</var>) in <var>object</var>, run these substeps:
 
   <ol>
-   <li><p>Set <var>header</var>'s key to <var>header</var>'s key,
-   <a spec=webidl>converted to ByteString</a>.
-   Rethrow any exception.
-
    <li><p><a lt=append for=Headers>Append</a>
-   <var>header</var>'s key/<var>header</var>'s value to
+   <var>key</var>/<var>value</var> to
    <var>headers</var>. Rethrow any exception.
```

Comment:
Since this is now the only substep you should just inline it in the paragraph that introduces it. "then for each mapping ... in object, append ..."